app/controllers/oauth/authorized_applications_controller.rb

```diff
@@ -15,6 +15,10 @@ class Oauth::AuthorizedApplicationsController < Doorkeeper::AuthorizedApplicatio
 
   include Localized
 
+  def index
+    @applications = OauthAuthorizedApplications.includes(:access_tokens).order(:created_at)
```

Comment:
Thanks to you for helping to make a better contribution.

Reply:
A better, more "doorkeeper" way to do this would be:

```
      @applications = Doorkeeper.config.application_model.authorized_for(current_resource_owner).order(created_at: :asc)

      respond_to do |format|
        format.html
        format.json { render json: @applications, current_resource_owner: current_resource_owner }
      end
```

Though as there's an ID primary key, we can also do `order(id: :asc)`, which is effectively the same as `created_at: :asc` but uses an index.

You may also want to send this change upstream to Doorkeeper: https://github.com/doorkeeper-gem/doorkeeper/blob/main/app/controllers/doorkeeper/authorized_applications_controller.rb